server/task.go

```diff
@@ -503,6 +504,31 @@ func (s *Server) changeTaskStatusWithPatch(ctx context.Context, task *api.Task,
 		}
 	}
 
+	// We will create an PITR activity when cutover stage done.
```

Comment:
IMO, it's better to move this block to `server/task_executor_pitr_cutover.go`, because `changeTaskStatusWithPatch` is for handling general cases of task status change. Whereas this if condition is tightly coupled with the PITR cutover task.